include/sancus_support/sancus_step.h

```diff
@@ -110,7 +119,9 @@ void timerA_isr_entry2(void)                                                   \
         "i"(TACTL_ENABLE),                                           /* %10 */ \
         "i"(TACCTL_ENABLE_CONT),                                     /* %11 */ \
         "m"(TACCTL0),                                                /* %12 */ \
-        "i"(TACTL_ENABLE_CONT)                                       /* %13 */ \
+        "i"(TACTL_ENABLE_CONT),                                      /* %13 */ \
+        "m"(__ss_dma_attacker_delay),                                /* %14 */ \
+        "m"(DMA_COUNTDOWN)                                           /* %15 */ \
```

Comment:
ideally we should move this inline asm to a dedicated `sancus-step.S` file at some point..